test/parallel/test-non-ascii.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const fs = require('node:fs');
+const common = require('../common');
```

Comment:
Common must be the first import

Reply:
sorry for the lint error. 
I've fixed it.